echopype/echodata/convention/1.0.yml

```diff
@@ -30,7 +30,7 @@ groups:
     ep_group: Provenance
   sonar:
     name: Sonar
-    description: contains specific metadata for the sonar system.
+    description: contains sonar system metadata and sonar data beam groups.
```

Comment:
This seems too many "data" in there. How about remove the second "data," like below?
```suggestion
    description: contains sonar system metadata and sonar beam groups.
```

Reply:
This looks good to me. I will go ahead and put @emiliom as a reviewer too, since he was the one that proposed this description.

Reply:
Sounds good.

Reply:
I think it's fine either way, with or w/o the extra "data". I'll commit that suggestion, then merge this PR.